src/test/java/com/serenitydojo/datatypes/WhenWorkingWithFloatingPointNumbers.java

```diff
@@ -8,16 +8,18 @@
 public class WhenWorkingWithFloatingPointNumbers {
 
     //
-    // Exercise 1: Convert 27 degrees Celcius to Farenheit
-    // The equation is: farenheit = (9 / 5) * celcius + 32
+    // Exercise 1: Convert 27 degrees Celsius to Fahrenheit
+    // The equation is: fahrenheit = (9 / 5) * celsius + 32
     //
     @Test
     public void convertToFarenheit() {
         double celcius = 27.0d;
         double farenheit = 0.0d;
 
         // TODO: Use a floating point calculation to calculate the farenheit equivalent of the celcius value.
-
+        //Ash: Expressed the elements of division as Doubles instead of integer type suggested in the provided equation. The integers were resulting in a value of 1 instead of 1.8 resulting in
+        //the fahrenheit value to evaluate inaccurately to 59-something instead of the expected value 80.6
+        farenheit = (9.0 / 5.0 * celcius) + 32;
```

Comment:
@wakaleo another place where I needed to manipulate the provided code. In this case, I expressed the constants in the formula Double instead of Integer to get the expected result. What are your thoughts?

I also wrote about it in Discord just yesterday https://discordapp.com/channels/912657814705553418/928292019913261127/1159295584960782396